website/docs/r/cis_global_load_balancer.html.markdown

```diff
@@ -70,6 +71,7 @@ Review the argument references that you can specify for your resource.
   - `region` - (Required, String) Enter a region code. Should not specify the multiple entries with the same region.
   - `pool_ids` - (Required, String) A list of pool IDs in failover priority for the provided region.
 - `session_affinity` - (Optional, String) Associates all requests from an end-user with a single origin. IBM sets a cookie on the initial response to the client, so that the consequent requests with the cookie in the request use the same origin, as long as it is available.
+- `steering_policy` - (Required, String) Steering Policy which allows off,geo,random,dynamic_latency.
```

Comment:
Please make sure it is inline with schema.. 
In schema it is optional..
Please make this argument optional and add default - "random"

Reply:
changed